mlflow/odahuflow/trainer/helpers/mlflow_helper.py

```diff
@@ -258,37 +258,47 @@ def _remember_cwd():
 
 
 def mlflow_to_gppi_cli():
-    parser = argparse.ArgumentParser()
 
-    def dir_type(string):
-        os.makedirs(string, exist_ok=True)
+    def make_dir(string):
+        try:
+            os.makedirs(string, exist_ok=True)
+        except FileExistsError as error:
+            raise ValueError(f'A file already exists with the same name: {string}\n'
+                             'Rename file or directory and try again.') from error
+
+    def validate_dir(string):
         if not os.path.isdir(string):
             raise ValueError(f'{string} is not a valid directory path')
-        return string
+
+    parser = argparse.ArgumentParser(description='Converts MLFLow model to GPPI.')
 
     parser.add_argument('--verbose', action='store_true', help='More extensive logging')
     parser.add_argument('--model-name', type=str, required=True, help='Name of GPPI Model')
     parser.add_argument('--model-version', type=str, required=True, help='Version of GPPI Model')
-    parser.add_argument('--mlflow-model-path', '--mlflow', required=True, type=dir_type,
-                        help='Path to source MLFlow model directory')
+    parser.add_argument('--mlflow-model-path', '--mlflow', required=True,
+                        type=str, help='Path to source MLFlow model directory')
```

Comment:
Why not using `dir_type`? Don't understand the point here.

Reply:
I just changed the type for `--gppi` flag and separated `dir_type` function into two (`make_dir` and `validate_dir`).
And, in the same way, I decided to change the type of `--mlflow` into `str` so it wouldn't be unique among other parameters.

But, I checked and now see that it gives a more neat error and returns errors faster.

```
: error: argument --mlflow-model-path/--mlflow: invalid dir_type value: '/Users/bohdan/wine_' 
```

So changed back to `dir_type` function.

Hence, for now, no point.)